src/ServiceControl.AcceptanceTesting/NServiceBusAcceptanceTest.cs

```diff
@@ -35,5 +39,34 @@ public void SetUp()
                 return testName + "." + endpointBuilder;
             };
         }
+
+        [TearDown]
+        public void TearDown()
+        {
+            if (!TestExecutionContext.CurrentContext.TryGetRunDescriptor(out var runDescriptor))
+            {
+                return;
+            }
+
+            var scenarioContext = runDescriptor.ScenarioContext;
+
+            TestContext.Out.WriteLine($@"Test settings:
+{string.Join(Environment.NewLine, runDescriptor.Settings.Select(setting => $"   {setting.Key}: {setting.Value}"))}");
+
+            TestContext.Out.WriteLine($@"Context:
+{string.Join(Environment.NewLine, scenarioContext.GetType().GetProperties().Select(p => $"{p.Name} = {p.GetValue(scenarioContext, null)}"))}");
+
+            if (TestExecutionContext.CurrentContext.CurrentResult.ResultState == ResultState.Failure || TestExecutionContext.CurrentContext.CurrentResult.ResultState == ResultState.Error)
+            {
+                TestContext.Out.WriteLine(string.Empty);
+                TestContext.Out.WriteLine($"Log entries (log level: {scenarioContext.LogLevel}):");
+                TestContext.Out.WriteLine("--- Start log entries ---------------------------------------------------");
+                foreach (var logEntry in scenarioContext.Logs)
+                {
+                    TestContext.Out.WriteLine($"{logEntry.Timestamp:T} {logEntry.Level} {logEntry.Endpoint ?? TestContext.CurrentContext.Test.Name}: {logEntry.Message}");
+                }
+                TestContext.Out.WriteLine("--- End log entries ---------------------------------------------------");
+            }
```

Comment:
Given that SC seems to have its own mechanism of printing logs, I think we can remove this part (as first imperfect step towards better debugging ability)